build.iotivity.config

```diff
@@ -0,0 +1,129 @@
+{
```

Comment:
There are two way to provide this build option . 
 1. integrate this file on the exist one
 2. make developer use this option as a parameter on the build.
I prefer second one, we can specify with external-static-lib or compile with parameter on using  tools/build.py. Why don't you add this guide on  the API document?